Server/tests/test_core_infrastructure_characterization.py

```
@@ -0,0 +1,1334 @@
+
+    def test_decorator_logs_any_exception_type(self, caplog_fixture):
+        """Verify decorator handles all exception types."""
+        caplog_fixture.clear()
+
+        @log_execution("any_exc", "AnyExc")
+        def func_raises_custom():
+            class CustomError(Exception):
+                pass
+            raise CustomError("Custom")
+
+        with pytest.raises(Exception):
```

Ruff (0.14.14)

Do not assert blind exception: `Exception`

(B017)